bulldozer/config_v1.go

```diff
@@ -60,8 +62,10 @@ type SquashOptions struct {
 }
 
 type UpdateConfig struct {
-	Whitelist Signals `yaml:"whitelist"`
-	Blacklist Signals `yaml:"blacklist"`
+	Allowlist Signals  `yaml:"allowlist"`
+	Denylist  Signals  `yaml:"denylist"`
+	Blacklist *Signals `yaml:"blacklist,omitempty"`
+	Whitelist *Signals `yaml:"whitelist,omitempty"`
```

Comment:
If you stick with this approach instead of a custom unmarshal function, let's add comments indicating these fields are deprecated and will be removed in a future v2 format.